src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -645,19 +645,25 @@ export default class AircraftCommander {
         let taxiDestination = data[0];
         const isDeparture = aircraft.category === FLIGHT_CATEGORY.DEPARTURE;
         const flightPhase = aircraft.flightPhase;
+        const airport = AirportController.airport_get();
 
         // Set the runway to taxi to
         if (!taxiDestination) {
-            const airport = AirportController.airport_get();
             taxiDestination = airport.departureRunwayModel.name;
         }
 
-        const runway = AirportController.airport_get().getRunway(taxiDestination.toUpperCase());
+        const runway = airport.getRunway(taxiDestination.toUpperCase());
```

Comment:
The `.toUpperCase()` should be handled by `.getRunway()`. If it isn’t there already, please add it there.

Reply:
`.getRunway()` actually gets the runway _model_, not the name, so we don't want to try to do a `.toUpperCase()` on the object. Thus, this should instead be `taxiDestination.name.toUpperCase()`